### PR TITLE
Search optimization

### PR DESCRIPTION
**Pull request by tgunnoe**
Let me know what should be fixed for the PR.

### CODE REVIEW

**Comment by ZachTRice** [comment]
The row sizes are off. The width of each row is much wider than before and the i button is not absolutely positioned.

I don't see any ie or edge specific issues.

**Comment by ZachTRice** [comment]
Examples of what I am seeing in my (Windows) browsers:
Edge 14
edge14_noscrollbar
IE11
ie11_noscrollbar
FireFox
ff_hasscrollbar
Chrome
chrome_hasscrollbar

**Comment by tgunnoe** [comment]
@ZachTRice what do you mean by the rows are much wider than before?  Does the 'i' button need to be absolutely positioned?

**Comment by localjo** [comment]
I figured out that the scrolling bug is caused by this old code in Worldview: https://github.com/nasa-gibs/worldview/blob/master/web/js/util/wv.util.browser.js#L444-L449

It was added three years ago in this commit: https://github.com/nasa-gibs/worldview/commit/fe8f0fb8c74fc1fa8ffc3257208912dfd1cd3545

So it looks like we'll want to test how changing it affects the time wheel.

**Comment by Benjaki2** [comment]
@localjo we could also add the `scrollable` class to the div to avoid unwanted side effects. But then we might run into this bug in the future. need to document things. It's a double edge sword.

File: web/js/proj/wv.proj.model.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
  * This code was originally developed at NASA/Goddard Space Flight Center for
  * the Earth Science Data and Information System (ESDIS) project.
  *
- * Copyright (C) 2013 - 2016 United States Government as represented by the
+ * Copyright (C) 2013-2017 United States Government as represented by the
  * Administrator of the National Aeronautics and Space Administration.
  * All Rights Reserved.
  */
```
[the remark this answers is not in the document]
Comment (localjo) [reply web/js/proj/wv.proj.model.js]: I'm going to address this in a separate issue: https://github.com/nasa-gibs/worldview/issues/565

**Pull request merged**